recipes/pymmcore-widgets/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "pymmcore-widgets" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymmcore-widgets-{{ version }}.tar.gz
+  sha256: c7886a0953d630f871771f5671caf605aa2ec4c5d328e25ba6e631b9eb559e24
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - fonticon-materialdesignicons6
+    - pymmcore-plus >=0.6.3
+    - python >=3.8
+    - qtpy
+    - superqt >=0.3.1
+    - useq-schema >=0.1.2
```

```suggestion
    - useq-schema >=0.1.2
    - vispy
```

Do you want to add `vispy`? It's [needed](https://github.com/pymmcore-plus/pymmcore-widgets/blob/352b7e9a7bf92080d53ac8f09307a8a4f30a48a0/src/pymmcore_widgets/_image_widget.py#L41) in some modules, apparently.

---

```suggestion
    - useq-schema >=0.1.2
    - numpy
```

And [same](https://github.com/pymmcore-plus/pymmcore-widgets/blob/352b7e9a7bf92080d53ac8f09307a8a4f30a48a0/src/pymmcore_widgets/_image_widget.py#L12) for numpy.

---

No thanks on both accounts.  Vispy is an extra needed for only one widget and a heavy enough dependency that id rather require users to have to conda install it explicitly.  And numpy is only needed for a type hint (for a getter function from a subdependency), it only gets imported inside of type checking blocks